versions/1.1.0-dev.md

```diff
@@ -134,6 +135,31 @@ The properties of the `update` or `copy` object MUST be compatible with the targ
 
 This object MAY be extended with [Specification Extensions](#specification-extensions).
 
+### Parameter Object
+
+The object represents parameters that can be used with actions.
+
+#### Fixed Fields
+
+| Field Name | Type | Description |
+| ---- | :----: | ---- |
+| <a name="parameter-name"></a>name | `string` | **REQUIRED** A name for the parameter that is unique for the action the parameter is defined. The name MUST match the following regular expression `[a-zA-Z-0-9]+`. |
+| <a name="parameter-default-values"></a>defaultValues | `string[] or Record<string, string>[]` | The values to use in case the environment variable is not defined. MAY contain zero or more entries. When values contains multiple entries, the action is effectively projected for each of the entries of the parameter. If multiple parameters are defined for any given action, parameters effectively act as a matrix definition and the action should be projected across each dimension. |
```

Comment:
```suggestion
| <a name="parameter-default-values"></a>defaultValues | [`string`] or [Map(`string`, `string`] | The values to use in case the environment variable is not defined. MAY contain zero or more entries. When values contains multiple entries, the action is effectively projected for each of the entries of the parameter. If multiple parameters are defined for any given action, parameters effectively act as a matrix definition and the action should be projected across each dimension. |
```